src/main/java/week2/BTree.java

```diff
@@ -0,0 +1,113 @@
+package week2;
+
+
+class BTree<T extends Comparable<T>> {
+
+  private boolean bothFound = false;
+  private Node<T> root;
+
+  BTree(Node<T> root) {
+    this.root = root;
+  }
+
+
+  /* Question 1: Given a Binary Tree and a key, write a function that prints all the ancestors
```

Comment:
Nit: https://google.github.io/styleguide/javaguide.html#s4.8.6-comments - put "*" on the start of lines within the comment.